awd/helpers.py

```diff
@@ -0,0 +1,255 @@
+from __future__ import annotations
+
+import json
+import logging
+from email.mime.application import MIMEApplication
+from email.mime.multipart import MIMEMultipart
+from typing import TYPE_CHECKING, Any
+
+import requests
+import smart_open
+from boto3 import client
+
+if TYPE_CHECKING:
+    from collections.abc import Iterator, Mapping
+
+    from mypy_boto3_s3.type_defs import PutObjectOutputTypeDef
+    from mypy_boto3_ses.type_defs import SendRawEmailResponseTypeDef
+    from mypy_boto3_sqs.type_defs import (
+        EmptyResponseMetadataTypeDef,
+        MessageAttributeValueTypeDef,
+        MessageTypeDef,
+        SendMessageResultTypeDef,
+    )
+
+
+logger = logging.getLogger(__name__)
+
+WILEY_HEADERS = {
+    "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 "
+    "(KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36"
+}
+
+
+class S3:
```

Comment:
I think it had come up at some point, and **not** a requested change here, but names like `S3Client` or even `S3ArticleClient` (if that's more accurate) could be helpful.  

Given this is a refactor, think it's reasonable to port along the previous names.  But seeing something like `S3` or `SQS`, it's unclear what the nature of those classes are.

Reply:
Good point about the class names, I'll update in the next PR